README.md

```diff
@@ -32,7 +32,7 @@ $ redis-server /usr/local/etc/redis.conf
 
 ## Localization
 
-_Coming soon_ (see [#57](https://github.com/mozilla/send/issues/57))
+Firefox Send localizations are managed via [Pontoon](https://pontoon.mozilla.org/projects/test-pilot-firefox-send/).
```

Comment:
I have a personal preference for "localization is managed", but plural works too. The badge is nice, but not sure it's effective (you would need to know what Pontoon or l20n are).

Maybe worth adding a bit more of information:

```
Firefox Send localization is managed via [Pontoon](https://pontoon.mozilla.org/projects/test-pilot-firefox-send/), not direct pull requests to the repository. If you want to fix a typo, add a new language, or simply know more about localization, please get in touch with the [existing localization team](https://pontoon.mozilla.org/teams/) for your language, or Mozilla’s [l10n-drivers](https://wiki.mozilla.org/L10n:Mozilla_Team#Mozilla_Corporation) for guidance.
```